### PR TITLE
s390: Fix wrong IP-address matching on zKVM upgrades 

### PR DESCRIPTION
**Pull request by Soulofdestiny**

### CODE REVIEW

**Comment by Soulofdestiny** [comment]
verification run: http://opeth.suse.de/tests/4875

File: tests/installation/set_static_ip.pm [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: set static ip for s390x
```
Comment (okurz) [inline tests/installation/set_static_ip.pm]: stating the obvious, why?

File: tests/installation/set_static_ip.pm [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+sub run() {
+    select_console 'install-shell';
+
+    my $ip    = get_var('VIRSH_GUEST');
+    my $ifcfg = "/mnt/etc/sysconfig/network/ifcfg-eth0";
+
+    script_run("cat $ifcfg");
+    script_run("sed -i -e \"/IPADDR=.*\$/s\@\@IPADDR=\'10.161.145.7/20\'\@\" $ifcfg");
+    script_run("cat $ifcfg | grep $ip") || die "IP address was not changed";
```
Comment (okurz) [inline tests/installation/set_static_ip.pm]: how about `assert_script_run(…, fail_message => 'IP address was not changed');`

**Comment by Soulofdestiny** [comment]
updated,
verification run: http://opeth.suse.de/tests/4876 (still running)

**Comment by okurz** [comment]
verification run looks good

**Pull request merged**